protocol/protobuf/pairing.proto

```diff
@@ -35,6 +35,8 @@ message Backup {
   FetchingBackedUpDataDetails watchOnlyAccountDetails = 14;
   repeated SyncChat chats = 15;
   FetchingBackedUpDataDetails chatsDetails = 16;
+  repeated SyncSetting settings = 17;
+  repeated SyncAccount watchOnlyAccounts = 18;
```

Comment:
I think when this feature is all done, we can remove the old "single" properties or at least deprecate them

Reply:
Yeah keeping both doesn't make sense 💯 
But I think it might better to do it right away, because it's not really related to the local backup feature 🤔